vega/static/vega-lite.js

```diff
@@ -2,6 +2,10 @@ var spec = {spec};
 var selector = "{selector}";
 var type = "{type}";
 
+console.log(spec);
+console.log(selector);
+console.log(type);
+
```

Comment:
They are really helpful for now.  I agree that some message in the console
would be helpful, but I probably wouldn't log the entire spec with the data
though. Maybe just the non-data part of the spec?

On Tue, May 17, 2016 at 4:20 PM, Jake Vanderplas <notifications@github.com>
wrote:

> In vega/static/vega-lite.js
> <https://github.com/vega/ipyvega/pull/32#discussion_r63621113>:
>
> > @@ -2,6 +2,10 @@ var spec = {spec};
> >  var selector = "{selector}";
> >  var type = "{type}";
> >
> > +console.log(spec);
> > +console.log(selector);
> > +console.log(type);
> > +
>
> Do you mean for these to stay in here? Some sort of permanent console
> message might actually be useful to keep.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/ipyvega/pull/32/files/d4541c081fe73ace7618ab34c6921f85cc63d86b..15e0a6819c108f72b5ef3dc647846de13bea216d#r63621113>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com